jiant/models.py

```diff
@@ -152,8 +153,14 @@ def build_sent_encoder(args, vocab, d_emb, tasks, embedder, cove_layer):
     elif any(isinstance(task, LanguageModelingTask) for task in tasks) or args.sent_enc == "bilm":
         assert_for_log(args.sent_enc in ["rnn", "bilm"], "Only RNNLM supported!")
         assert_for_log(
-            args.input_module != "elmo" and not args.input_module.startswith("bert"),
-            "LM with full ELMo and BERT not supported",
+            not (
+                args.input_module == "elmo"
+                or args.input_module.startswith("bert")
```

Comment:
`or input_module_uses_pytorch_transformers` ?

Reply:
Specific to these two models. Once we add LMs like GPT/GPT2/TransXL, this logic will stay correct, but input_module_uses_pytorch_transformers won't.